example-apps/onedrive/utils.js

```diff
@@ -8,7 +8,7 @@ const contentDisposition = require('content-disposition')
 // on the OneDrive API. We intentionally don't set it up as an `afterResposne`
 // handler because not *all* calls need it (i.e. the auth test and file create),
 // so we break it out and share the code this way instead.
-const parseResponse = (type, response) => {
+const parseResponse = (z, type, response) => {
```

Comment:
Notice that the signature of this function has changed. All usage has been updated.